tests/views/tp.py

```diff
@@ -0,0 +1,218 @@
+# -*- coding: utf-8 -*-
```

Comment:
We are using `translation_project` instead of `tp` in module names. As I understood a short version was intended here.